src/System.Private.CoreLib/src/System/Enum.CoreCLR.cs

```diff
@@ -95,7 +96,7 @@ public bool HasFlag(Enum flag)
             return InternalHasFlag(flag);
         }
 
-        public static string GetName(Type enumType, object value)
+        public static string GetName(Type enumType, object? value)
```

Comment:
Why is `value` defined as `object?` rather than `object`?  Are there implementations of `Enum` where `value` is allowed to be null?

Reply:
don't know why i missed this :(